internal/gps/strip_vendor.go

```diff
@@ -22,11 +22,7 @@ func stripVendor(path string, info os.FileInfo, err error) error {
 		}
 
 		if (info.Mode() & os.ModeSymlink) != 0 {
-			realInfo, err := os.Stat(path)
-			if err != nil {
```

Comment:
Rather than simply ignoring errors here, let's verify that it's an error indicating that the target file does not exist.

(I'm still thinking about this change overall, but if we do go this direction, then we absolutely need this check.)

Reply:
I guess adding err != nil { do_nothing } is not necessary. it ignores when lstat() succeed but stat() is either returns null or non-directory. What checks would you like to add specifically?
This is a case our whole team had to rollback.